### PR TITLE
Add testcase to ensure ratelimits are working correctly

### PR DESCRIPTION
**Pull request by MustafaSaber**
Signed-off-by: Mustafa Abdelrahman <mustafa.abdelrahman@zalando.de>

### CODE REVIEW

File: test/e2e/routegroup.go [hunk 1]
```diff
@@ -313,6 +313,88 @@ rBackend4: Path("/router-response") -> inlineContent("NOT OK") -> <shunt>;
...
+		framework.ExpectNoError(e2epod.WaitForPodNameRunningInNamespace(f.ClientSet, pod.Name, pod.Namespace))
+
+		// RouteGroup
+		By("Creating a routegroup with name " + serviceName + " in namespace " + ns + " with hostname " + hostName)
+		rg := createRouteGroup(serviceName, hostName, ns, labels, nil, port, rgv1.RouteGroupRouteSpec{
+			PathSubtree: "/backend",
+			Methods:     []rgv1.HTTPMethod{rgv1.MethodGet},
+			Predicates:  []string{},
+			Filters: []string{
+				`status(201)`,
```
Comment (szuecs) [inline test/e2e/routegroup.go]: The status code you should set in the backend not the RouteGroup

File: test/e2e/routegroup.go [hunk 2]
```diff
@@ -313,6 +313,88 @@ rBackend4: Path("/router-response") -> inlineContent("NOT OK") -> <shunt>;
...
+				`clusterRatelimit("foo", 1, "1s")`,
```
Comment (szuecs) [inline test/e2e/routegroup.go]: Not sure if 1s will be too flaky in e2e tests, maybe 5s or so?

**Review by szuecs — CHANGES_REQUESTED** [review]
Small changes that should enhance the Test

File: test/e2e/routegroup.go (hunk 2 above)
Comment (MustafaSaber) [reply test/e2e/routegroup.go]: I was going with trail and error here, but yes maybe 5 would be better start

**Comment by szuecs** [comment]
:+1:

**Comment by linki** [comment]
👍

**Review by szuecs — APPROVED** [review]

**Pull request merged**